gems/pending/VMwareWebService/MiqVimEventMonitor.rb

```diff
@@ -2,11 +2,12 @@
 require 'VMwareWebService/MiqVimInventory'
 
 class MiqVimEventMonitor < MiqVimInventory
-  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100)
+  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100, maxWait = 60)
```

Comment:
@agrare I just noticed that the caller will always pass a value for pgSize (probably nil because it's not in the config), so this value of 100 is never realized.  What is that going to do, performance wise?